src/thorin/pass/pass.cpp

```diff
@@ -39,7 +39,7 @@ void PassMan::pop_states(size_t undo) {
 }
 
 void PassMan::run() {
-    world().ILOG("run");
+    world().verify().ILOG("run");
```

Comment:
Are there cases when we don't want to verify before ILOG?

Reply:
There may be cases where you are in the middle of rebuilding sth where an external gets some free variables before you mark it as internal.